### PR TITLE
Service version endpoint

### PR DESCRIPTION
**Pull request by danielSanchezQ**
I added some basic unittest.  I had to modify the server bootstrapper in tests so it has some default value and the server launches without problems but probably It will need some changes for testing also that the server reports a correct feedback code when it cannot load the env variable.
@dkijania I can take care of it later or feel free to take account of it too as you need 😄

### CODE REVIEW

**Review by eugene-babichenko — APPROVED** [review]
LGTM

File: vit-servicing-station-lib/src/v0/context.rs [hunk 1]
```diff
@@ -50,11 +54,16 @@ pub mod test {
         let db_url = format!("file:{}?mode=memory&cache=shared", name);
         let pool = db::load_db_connection_pool(&db_url).unwrap();
         let block0: Vec<u8> = vec![1, 2, 3, 4, 5];
-        Arc::new(RwLock::new(Context::new(pool, "", block0)))
+        Arc::new(RwLock::new(Context::new(
+            pool,
+            "",
+            block0,
+            "2.0".to_string(),
```
Comment (andrcmdr) [inline vit-servicing-station-lib/src/v0/context.rs:61]: We definitely need to have an agreement about versioning the VITss API

File: vit-servicing-station-lib/src/v0/context.rs [hunk 2]
```diff
@@ -50,11 +54,16 @@ pub mod test {
...
+        )))
     }
 
     pub fn new_test_shared_context(db_url: &str, block0_path: &str) -> SharedContext {
         let pool = db::load_db_connection_pool(db_url).unwrap();
-        new_shared_context(pool, block0_path)
+        new_shared_context(pool, block0_path, "2.0".to_string())
```
Comment (andrcmdr) [inline vit-servicing-station-lib/src/v0/context.rs:67]: We definitely need to have an agreement about versioning the VITss API and VITss as well (i.e. we will implement frequent tagging in CI for VITss repo), so there will be no need to hardcode the versioning in tests in near perspective.

**Review by Zeegomo — APPROVED** [review]

File: vit-servicing-station-lib/src/v0/endpoints/service_version/handlers.rs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            .and(warp::get())
+            .and(with_context)
+            .and_then(service_version);
+
+        let result = warp::test::request().method("GET").reply(&filter).await;
+        assert_eq!(result.status(), warp::http::StatusCode::OK);
+        println!("{}", String::from_utf8(result.body().to_vec()).unwrap());
+        let service_version_result: ServiceVersion =
+            serde_json::from_str(&String::from_utf8(result.body().to_vec()).unwrap()).unwrap();
+        assert_eq!(service_version_result.service_version, "2.0".to_string());
```
Comment (andrcmdr) [inline vit-servicing-station-lib/src/v0/endpoints/service_version/handlers.rs:35]: We definitely need to have an agreement about versioning the VITss API and VITss as well (i.e. we will implement frequent tagging in CI for VITss repo), so there will be no need to hardcode the versioning in tests and asserts in code in near perspective.

File: vit-servicing-station-tests/src/common/clients/rest.rs [hunk 4]
```diff
@@ -255,6 +269,10 @@ impl RestPathBuilder {
         self.path("health")
     }
 
+    pub fn service_version(&self) -> String {
```
Comment (andrcmdr) [inline vit-servicing-station-tests/src/common/clients/rest.rs:272]: Notes: the exact place of entry point for versioning endpoint for client side app.

File: vit-servicing-station-tests/src/tests/rest/service_version.rs [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+};
+use assert_fs::TempDir;
+
+#[test]
+pub fn service_version() {
+    let temp_dir = TempDir::new().unwrap();
+    let (hash, token) = data::token();
+
+    let db_path = DbBuilder::new().with_token(token).build(&temp_dir).unwrap();
+    let version = "TestV1".to_string();
```
Comment (andrcmdr) [inline vit-servicing-station-tests/src/tests/rest/service_version.rs:13]: We definitely need to have an agreement about versioning the VITss API and VITss as well (i.e. we will implement frequent tagging in CI for VITss repo), so there will be no need to hardcode the versioning in tests in near perspective.

**Review by andrcmdr — APPROVED** [review]
We definitely need to have an agreement about versioning the VITss API and VITss whole codebase as well (i.e. we will implement frequent tagging in CI for VITss repo), so there will be no need to hardcode the versioning in tests and assertion points in code in near perspective.

**Pull request merged**